crates/rbuilder/src/bin/relay-sender.rs

```diff
@@ -1,8 +1,8 @@
+//! This app is intended to be used for testing against a fake-relay I made (https://github.com/ZanCorDX/fake-relay) without needing all the builder or relay running
```

Comment:
- I'd remove "I made"
- "needing all the builder.." -> "needing to have the builder.."